src/tools/python/datatypes/lists.md

```diff
@@ -179,6 +184,41 @@ print(5 in a) # => False
 print(0 in a) # => False
 ```
 
+List `=`
+---
+
+So far we've seen that the assignment statement `=` copies the contents of one variable into the other. This is true for ints, strings, bools, and floats.
+
+However, variables for lists store a pointer to where we can find the actual list in memory, and the `=` statement changes the pointer as opposed to copying the list's contents. The following example illustrates this:
```

Comment:
explanation of pointers may need more work / diagrams
I'd also put `pointer` instead of pointer. Typically I reserve inline code blocks for any terms that are programming related

Saying "pointer in memory" may be vague for someone new to programming.